samples/sensor/mcux_acmp/src/main.c

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2020 Vestas Wind Systems A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <device.h>
+#include <drivers/sensor.h>
+#include <drivers/sensor/mcux_acmp.h>
+#include <zephyr.h>
+
+#include <stdio.h>
+
+#ifdef CONFIG_BOARD_TWR_KE18F
+#define ACMP_NODE  DT_NODELABEL(cmp2)
+#define ACMP_LABEL DT_LABEL(ACMP_NODE)
+#define ACMP_POSITIVE 5
+#define ACMP_NEGATIVE 5
+#define ACMP_DAC_VREF 0
+#else
+#error Unsupported board
```

Comment:
The usual thing to do would be to rely on a sample-specific alias. Why not do that here and avoid a proliferation of `#ifdef CONFIG_BOARD_...`?

Reply:
@mbolivar-nordic Thanks for reviewing! There are other, board-specific configurations needed as well (as shown in the quoted code above). I do not predict this sample to become enabled for a wide variety of boards, so this list of `#ifdefs` should not become too long.